src/Components/Pages/Home.js

```diff
@@ -1,8 +1,16 @@
 import RTPriceFormattedTable from "../Tables/RTPriceFormattedTable";
```

Comment:
Kinda different from what we have on the Main branch. Make sure you do a diff and include the code we have at HEAD of Main.

Reply:
Sounds good